Procurement/ViewModel/SetBuyoutViewModel.cs

```diff
@@ -69,6 +79,38 @@ public void SetBuyoutInfo(ItemTradeInfo info)
             offerInfo.Update(info.CurrentOffer);
             priceInfo.Update(info.Price);
             Notes = info.Notes;
-        }        
+        }
+
+        public bool IsDataInClipboard
+        {
+            get { return isDataInClipboard; }
+            set
+            {
+                isDataInClipboard = value;
+                OnPropertyChanged();
+            }
+        }
+
+        public ICommand PobDataGenerationCommand => new RelayCommand(GetPobData, CanGetPobData);
+
+        private void GetPobData(object o)
+        {
+            try
+            {
+                Clipboard.SetText(item.PobData);
+
+                IsDataInClipboard = true;
+            }
+            catch (Exception e)
+            {
+                MessageBox.Show("Unable to generate POB item data, please check log.","Error", MessageBoxButton.OK, MessageBoxImage.Error);
```

Comment:
Add a space between the first and second arguments.  Use a semicolon instead of a comma in the text to avoid a comma splice error.